Makefile.toml

```diff
@@ -381,18 +381,11 @@ dependencies = ["build-variant"]
 script = [
 '''
 mkdir -p ${BUILDSYS_OUTPUT_DIR}/latest
-ln -snf ../${BUILDSYS_NAME_FULL}.img.lz4 \
-  ${BUILDSYS_OUTPUT_DIR}/latest/${BUILDSYS_NAME_VARIANT}.img.lz4
-ln -snf ../${BUILDSYS_NAME_FULL}-data.img.lz4 \
-  ${BUILDSYS_OUTPUT_DIR}/latest/${BUILDSYS_NAME_VARIANT}-data.img.lz4
-ln -snf ../${BUILDSYS_NAME_FULL}-boot.ext4.lz4 \
-  ${BUILDSYS_OUTPUT_DIR}/latest/${BUILDSYS_NAME_VARIANT}-boot.ext4.lz4
-ln -snf ../${BUILDSYS_NAME_FULL}-root.ext4.lz4 \
-  ${BUILDSYS_OUTPUT_DIR}/latest/${BUILDSYS_NAME_VARIANT}-root.ext4.lz4
-ln -snf ../${BUILDSYS_NAME_FULL}-root.verity.lz4 \
-  ${BUILDSYS_OUTPUT_DIR}/latest/${BUILDSYS_NAME_VARIANT}-root.verity.lz4
-ln -snf ../${BUILDSYS_NAME_FULL}-migrations.tar \
-  ${BUILDSYS_OUTPUT_DIR}/latest/${BUILDSYS_NAME_VARIANT}-migrations.tar
+for artifact in ${BUILDSYS_OUTPUT_DIR}/${BUILDSYS_NAME_FULL}*; do
```

Comment:
One pitfall of using a loop here is if a user builds a variant as an image, and then later rebuilds that image as a vmdk, if the build-id does not change it will link the old artifacts img.lz4 as well.

Reply:
That might be desirable in some situations, for example: to build both a raw disk image *and* a VMDK.

Reply:
I have a fix coming soon for #1027 that should address this; artifacts from a previous `buildsys` invocation will be cleaned up prior to the next, so we shouldn't hit the situation where we have both types at once.

Reply:
@bcressey  can correct me, but I think his solution is just for cleaning up old artifacts, such as in the situation I mentioned above. This loop would still be necessary for generating the proper symlinks under the build/images/latest directory.

Reply:
Ah yeah you're right!